Makefile.am

```diff
@@ -55,6 +55,33 @@ noinst_PROGRAMS =
 # Libraries that should be produced but not installed.
 noinst_LTLIBRARIES = 
 
+# Binaries that should be installed in libexecdir
+libexec_PROGRAMS = 
+
+substitute_libexecdir = sed \
+	-e 's|@libexecdir[@]|$(libexecdir)|g' \
+	$(NULL)
+
+# systemd service units
+#
+# Defined as systemdsystemunitdir in ../configure.ac.
```

Comment:
The relative path in this comment is now wrong since this file is in the same directory as `configure.ac`.